### DISCUSSION TITLE
release notes for v 2.4.3

### DISCUSSION THREAD

**Discussion by AndreiHushchyn** [Azure/azure-functions-durable-extension#1818] [Q&A] [answered]
Why there are no release notes for v 2.4.3 ?

**Comment by cgillum** [comment, marked as answer]
Thanks for bringing this up and sorry about the confusion. The v2.4.3 release was an out-of-band release to fix an internal engineering issue, and we forgot to formally publish a release (something we should consider automating 😅). I've gone ahead and published these release notes here: https://github.com/Azure/azure-functions-durable-extension/releases/tag/v2.4.3.
